desktop/apps/auction2/reducers.js

```diff
@@ -124,12 +126,21 @@ function auctionArtworks(state = initialState, action) {
           }
         }, state)
       } else if (artistId === 'artists-you-follow') {
-        return u({
-          filterParams: {
-            artist_ids: [],
-            include_artworks_by_followed_artists: true
-          }
-        }, state)
+        if (state.filterParams.include_artworks_by_followed_artists === true) {
+          return u({
+            filterParams: {
+              artist_ids: [],
+              include_artworks_by_followed_artists: false
+            }
+          }, state)
+        } else {
+          return u({
+            filterParams: {
+              artist_ids: [],
+              include_artworks_by_followed_artists: true
+            }
+          }, state)
+        }
```

Comment:
So as to reduce duplication, you can probably collapse this into a one-line conditional that evaluates `state.filterParams.include_artworks_by_followed_artists`, then pass the result into a single updeep at the bottom.